docs/develop/api-reference/index.mdx

```diff
@@ -228,78 +229,9 @@ Sets the value in cache with a given Time To Live (TTL) seconds. If a value for
 | value      | []Byte | The value to be stored.                       |
 | ttlSeconds | int    | Time to Live for the item in Cache.           |
 
-<SdkExamples
-  js={`
-const authToken="eyJhbGc.MyTestToken";
-const defaultTTL = 15;
-const momento = new SimpleCacheClient(authToken, defaultTtl);
-momento.set('test-cache', 'test-key', 'test-value');
-`}
-  python={`
-import momento.simple_cache_client as scc\n
-_MOMENTO_AUTH_TOKEN = "eyJhbGc.MyTestToken"
-_ITEM_DEFAULT_TTL_SECONDS = 15\n
-with scc.SimpleCacheClient(_MOMENTO_AUTH_TOKEN, _ITEM_DEFAULT_TTL_SECONDS) as cache_client:
-    cache_client.set('test-cache', 'test-key', 'test-value')
-`}
-  java={`
-String MOMENTO_AUTH_TOKEN = "eyJhbGc.MyTestToken";
-int DEFAULT_ITEM_TTL_SECONDS = 15;\n
-SimpleCacheClient simpleCacheClient = SimpleCacheClient
-    .builder(MOMENTO_AUTH_TOKEN, DEFAULT_ITEM_TTL_SECONDS)
-    .build()\n
-simpleCacheClient.set('test-cache', 'test-key', 'test-value');
-    `}
-  go={`
-const (
-    authToken             = "eyJhbGc.MyTestToken"
-    itemDefaultTtlSeconds = 15
-)
-client, err := momento.NewSimpleCacheClient(authToken, itemDefaultTtlSeconds)
-if err != nil {
-    panic(err)
-}
-_, err = client.Set(&CacheSetRequest{
-    CacheName: "test-cache",
-    Key:       "test-key",
-    Value:     "test-value",
-})
-    `}
-  csharp={`
-TimeSpan DEFAULT_TTL = TimeSpan.FromSeconds(15);
-ICredentialProvider authProvider = new StringMomentoTokenProvider("eyJhbGc.MyTestToken");
-using SimpleCacheClient client = new SimpleCacheClient(Configurations.Laptop.Latest(), authProvider, DEFAULT_TTL);
-await client.SetAsync("test-cache", "test-key", "test-value");
-    `}
-  rust={`
-let auth_token = "eyJhbGc.MyTestToken";
-let item_default_ttl_seconds = 15;
-let mut cache_client = SimpleCacheClientBuilder::new(
-    auth_token,
-    NonZeroU64::new(item_default_ttl_seconds).unwrap(),
-)
-.unwrap()
-.build();\n
-let cache_name = String::from("cache");
-let key = String::from("my_key");
-let value = String::from("my_value");
-cache_client
-    .set(&cache_name, key.clone(), value.clone(), None)
-    .await
-    .unwrap();
-    `}
-  ruby={`
-require 'momento'
-MOMENTO_AUTH_TOKEN = "eyJhbGc.MyTestToken"
-DEFAULT_TTL_SECONDS = 15
-client = Momento::SimpleCacheClient.new(
-  auth_token: MOMENTO_AUTH_TOKEN, default_ttl: DEFAULT_TTL_SECONDS
-)
-response = client.set('test-cache', 'test-key', 'test-value')
-raise response.error if response.error?
-  `}
-  cli={`momento cache set --key test-key --value test-value`}
-/>
+<SdkExampleSnippets
+  snippetId={'API_Set'}
+  />
```

Comment:
After some iteration - this is now what I'm thinking all of these tab-view examples will end up looking like in the mdx files.